docs/source/Regridding.rst

```diff
@@ -118,325 +55,330 @@ Required Arguments:
-      command :command:`gridspec-create latlon --help`.
+We will use the example of regridding the out-of-the-box
+:literal:`GEOSChem.Restart.20190701_0000z.c48.nc4` restart file from C48 to
+a C120 base resolution stretched grid with a stretch factor of 4.0 over Bermuda
+to demonstrate the stretched cubed-sphere regridding process:
 
-      Create a subdirectory named latlon and move all of your latlon grid
-      specification files there.
+#. Create a source grid specification using :code:`gridspec-create`:
 
-      .. code-block:: console
+   .. code-block:: console
 
-         $ gridspec-create latlon 90 180                # Generic 1 x 1 grid
-         $ gridspec-create latlon 46 72 -dc -pc -hp     # GEOS-Chem Classic 4 x 5
-         $ gridspec-create latlon 91 144 -dc -pc -hp    # GEOS-Chem Classic 2 x 2.5
-         $ gridspec-create latlon 361 576 -dc -pc -hp   # MERRA-2 0.5 x 0.625
-         $ gridspec-create latlon 721 1172 -dc -pc -hp  # GEOS-FP 0.25 x  0.3125
+      $ gridspec-create gcs 48
 
-         $ mkdir latlon
-         $ mv regular_lat_lon*.nc latlon
+   This will produce 7 files - :literal:`c48_gridspec.nc` and
+   :literal:`c48.tile[1-6].nc`
 
-#. (Optional) View contents of grid spec file:
+#. Create a target grid specification using :code:`gridspec-create`:
 
    .. code-block:: console
 
-      $ gridspec-dump c24/c24_gridspec.nc
+      $ gridspec-create sgcs 120 -s 4.0 -t 32.0 -64.0
 
-      ... etc. for other grids ...
+   Here, the :code:`-s` option denotes the stretch factor and the :code:`-t`
+   option denotes the latitude / longitude of the centre point of the grid
+   stretch. 
+   
+   Again, this will produce 7 files - :literal:`c120_..._gridspec.nc` and
+   :literal:`c120_..._tile[1-6].nc`, where :literal:`...` denotes randomly
+   generated characters.
 
-#. Initialize your GCPy conda environmnt (which includes ESMF as a
-   dependency):
+#. Create the regridding weights for the regridding transformation using
+   :code:`ESMF_RegridWeightGen`:
 
    .. code-block:: console
 
-      $ conda activate gcpy_env
+      $ ESMF_RegridWeightGen                 \
+          --source c48_gridspec.nc           \
+          --destination c120_..._gridspec.nc \
+          --method conserve                  \
+          --weight c48_to_c120_stretched_weights.nc 
+
+   This will produce a log file, :literal:`PET0.RegridWeightGen.Log`, and our
+   regridding weights, :literal:`c48_to_c120_stretched_weights.nc`
 
-#. Navigate to the directory that will store the regridding
-   weights. (Recall that we created this in created this in step #2.
+#. Finally, use the grid weights produced in step 3 to complete the regridding:
 
    .. code-block:: console
 
-      $ cd /path/to/RegridInfo/Weights
+      $ python -m gcpy.regrid_restart_file    \
+          GEOSChem.Restart.20190701_0000z.c48 \
+          c48_to_c120_stretched_weights.nc    \
+          GEOSChem.Restart.20190701_0000z.c48
```

Comment:
Shouldn't this also have stretched grid, target lat, and target lon passed? When I execute as it I don't get the stretch factor and target lat/lon global attributes.

Reply:
Yes, you're right. That's the whole point of the update - can't believe I missed that.